### ISSUE TITLE
Change weight used for computing loss function according to 

### ISSUE THREAD

**Issue by mjlbach** [mjlbach/CLUnet#8]
From Sadanandan et. al _Scientific Reports_ 2017: "To compensate for this imbalance we created a weighted label image, in addition to the label image. We set the weight in such a way that the weighted sum of the number of pixels belonging to one class was the same as that for the other class. This enabled the loss function to treat the foreground and background regions with equal importance. We forced the network to learn the features corresponding to the foreground regions by setting an additional weight of 3 to the foreground region. The choice of this parameter depended on the dataset, based on the density of cells in the image, and could be automatically determined based on the proportion of cells covering the training image. "

**Comment by mjlbach** [comment]
Also foreground and background should likely have loss functions computed for seperate feature dimensions.
